core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -178,7 +178,12 @@ abstract class FileCommitProtocol extends Logging {
    * implementation deletes the file immediately.
    */
   def deleteWithJob(fs: FileSystem, path: Path, recursive: Boolean): Boolean = {
-    fs.delete(path, recursive)
+    if (fs.getConf.getInt("fs.trash.interval", 0) > 0 &&
```

Comment:
Where can I find the document of `fs.trash.interval`? It's better to add a comment to link to the hadoop conf, to explain that this conf indicates enabling the trash feature.

Reply:
I had added  a comment to link hadoop conf doc page. Thanks.